mesh_led_control_example.cpp

```diff
@@ -192,8 +191,15 @@ static void init_socket()
 {
     my_socket = new UDPSocket(network_if);
     my_socket->set_blocking(false);
-    my_socket->bind(UDP_PORT);    
+    my_socket->bind(UDP_PORT);
     my_socket->setsockopt(SOCKET_IPPROTO_IPV6, SOCKET_IPV6_MULTICAST_HOPS, &multicast_hops, sizeof(multicast_hops));
+
+    ns_ipv6_mreq_t mreq;
+    memcpy(mreq.ipv6mr_multiaddr, multi_cast_addr, 16);
+    mreq.ipv6mr_interface = 0;
```

Comment:
Why zero? Can it change?

Reply:
Zero means "default". Best we can do as we don't have visibility of Nanostack interface IDs.

There is a PR going through to add a generic NSAPI for this - https://github.com/ARMmbed/mbed-os/pull/4820. That is intending to use interface address instead.